packages/react-components/src/components/Pagination/Pagination.jsx

```diff
@@ -105,17 +109,20 @@ class Pagination extends Component {
     let lastPage;
     if (showLastPage && currentPage < totalPages - maxPageListLength + 1) {
       lastPage = (
-        <span>
-          <a aria-label="...">...</a>
-          <a
+        <>
+          <button aria-label="..." type="button">
```

Comment:
This feels wrong as a button or anchor because it's just text.

Reply:
:thinking: Does it need to be a button or would a span work? When I click the `...` in storybook I don't see anything happen, so we may not need a clickable element.

Reply:
I brought it up because it seemed like it could result in a possible design change for the [web component ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/35098). We can update it to be a focusable span and apply the same styling (including when focused and hover) as the rest of the anchors (now buttons) to limit the scope of this PR. 

**Should it be focusable?** I can see an argument being made for screen readers. If it should be focusable, we should consider making the ellipsis visually different from the other buttons when focused/hovering. It feels wrong to me when something looks like a button but does nothing, especially next to real buttons that look exactly the same. We should also consider updating its aria-label value. Using VoiceOver, "..." isn't read by this screen reader and may not be providing the value we think it should.